strapi/strapi
# Confirmation message when deleting a transfer token or an API token from the list

### What does it do?

As a developer having access to the delete command for API tokens and transfer tokens, when I click on the delete command from the list of API tokens or transfer tokens, then I want a modal asking me for the confirmation of the operation.

### Why is it needed?

To prevent errors when a user try to delete a Token

### How to test it?

 - As a developer 
   - when I have the delete permission on API tokens
   - and I access the API tokens list
   - and I click on the delete button for one of the tokens in the list
     - then I want a Dialog Modal like in the design system (  ) asking for confirmation of the process
     - when I click on Confirm
       - then the token is deleted
     - when I click on Cancel
       - then the operation of deleting the token is dismissed
 - when I have the delete permission on transfer tokens
    - and I access the transfer tokens list
    - and I click on the delete button for one of the tokens in the list
      - then I want a Dialog Modal like in the design system (  ) asking for confirmation of the process
      - when I click on Confirm
         - then the token is deleted
      - when I click on Cancel
        - then the operation of deleting the token is dismissed
